src/views/GovernanceReviewTeam/Notes/index.tsx

```diff
@@ -36,10 +40,19 @@ const Notes = () => {
     await resetForm();
   };
 
+  useEffect(() => {
+    dispatch(fetchIntakeNotes(systemId));
+  }, [dispatch, systemId]);
+
   const initialValues = {
     note: ''
   };
 
+  const sortedNotes = notes.sort(
```

Comment:
Currently, the API returns notes oldest to newest. Rather than reversing the order or waiting for API changes to be made, I decided to make sure the notes are properly sorted.

Reply:
Yeah, I didn't want this to get blocked so I manually sorted it for now.